ms.html

```diff
@@ -26,6 +26,13 @@
             <img src="media/logo.jpg" alt="logo">
         </logoimg>
     </gm-header>
+    <div class="statusArea card row bg-light is-center">
```

Comment:
Why not just make the StatusCard a DEER component? You probably do not even need the nested ones separated.

Reply:
Just a choice made in a one day get it out the door move.  I kept each status individualized as I wasn't sure if each interface where they might be used would need/want them all together.  I needed another opinion.

If it would be better to package them all together in one StatusCard Template and that is something that will be reliably re-used, then that would be an improvement.